app/scripts/controllers/deployment.js

```diff
@@ -39,6 +39,7 @@ angular.module('openshiftConsole')
       }
     ];
 
+    $scope.isDeploymentConfig = ($routeParams.deploymentconfig) ? true : false;
```

Comment:
isDeploymentConfig isnt really right given how refer to things elsewhere in the code, its isDeployment, or maybe hasDeploymentConfig.